validator/sawtooth_validator/journal/block_store.py

```diff
@@ -85,9 +84,20 @@ def __init__(self, database):
             database.pointer,
             ctypes.byref(self.pointer))
 
-    def _get_data_by_id(self, object_id, ffi_fn_name):
+    def _get_data_by_num(self, object_id, ffi_fn_name):
         (vec_ptr, vec_len, vec_cap) = ffi.prepare_vec_result()
+        _libexec(
+            ffi_fn_name,
+            self.pointer,
+            ctypes.c_int(object_id),
```

Comment:
Is this the right sized value? Block numbers are `u64`

Reply:
Oh, darn.  I spotted that in my review and then didn't commit.  Updated to c_ulonglong.  I'll update the PR as soon as the build/test cycle is done.

Reply:
Since you're going to have to do a force push, could you also tidy up your commit message by following https://chris.beams.io/posts/git-commit/ ?

Reply:
Sure.  I can make it much shorter and more imperative.

Reply:
That's all ready.  Auto-Jenkins-buildn isn't working because I'm not whitelisted: I assume this will fix itself at some point.